CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ The changelog for `Superwall`. Also see the [releases](https://github.com/superw
 ## 1.1.7
 
 ### Enhancements
-- Adds `restore_start`, `restore_complete`, and `restore_fail` events.
+- Fixes `SW-2855` by adding `restore_start`, `restore_complete`, and `restore_fail` events.
+- Fixes `SW-2854` where abandoning transaction by pressing back would prevent restarting the transaction
```

Comment:
I think this one should be under a 
### Fixes

header. Whereas the other is an enhancement